api/RTCRemoteOutboundRtpStreamStats.json

```diff
@@ -145,55 +145,6 @@
             "deprecated": false
           }
         }
-      },
-      "reportsSent": {
```

Comment:
`RTCRemoteOutboundRtpStreamStats` is a dictionary and tricky to test, but I can't find any trace of this in Chromium, Gecko, or WebKit source outside of tests, so I think it should indeed be removed.